### PR TITLE
Completed feature #1 desktop.

### PR DESCRIPTION
**Pull request by VeraYeh**
Completed desktop first, please review, thanks.
image
image
image

### CODE REVIEW

File: client/pages/login-form.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    this.setState({
+      babyName: event.target.value
+    });
+  }
+
+  handlePassword(event) {
+    this.setState({
+      password: event.target.value
+    });
+  }
```
Comment (scbowler) [inline client/pages/login-form.jsx]: You can simplify this to one method with:
```JSX
handleChange(event) {
  const name = event.target.name;
  const value = event.target.value;
  
  this.setState({ [name]: value });
}
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good! Just one update with your form component, let me know if you have any questions.

File: client/pages/login-form.jsx (hunk 1 above)
Comment (VeraYeh) [reply client/pages/login-form.jsx]: @scbowler 
Thanks for the comment. Code updated.

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**